toaster_msgs/srv/AddFactsToAgent.srv

```diff
@@ -1,4 +0,0 @@
-toaster_msgs/Fact[] facts
-string agentId
----
-bool answer
```

Comment:
It compile without (I completly delete devel in my catkin_ws to be sure) so I think its ok :)